### DISCUSSION TITLE
Notification sound on tablet speaker while conected to a station

### DISCUSSION THREAD

**Discussion by dl1esz** [N0BOY/FT8CN#124] [Ideas]
Hi,
first of all tnx for this fine software allows me to use my FT818 an a 8" Tablet (Lenovo A7lite T225) during POTA or other outdoor akiity.
In a case of connecting to a station it would be fine to get a short notification sound ( like a pling) to the build in tablet speaker according the notifikation feature in JTDX.

tnx for this fine software

B.r. Stefan DL1ESZ

**Comment by PhilipG0ISW** [comment]
This is a great idea. 

I have been using the FT8CN app with my Xiegu X6200 for the past week on holiday QRP as EA6/G0ISW and really like it. Connection has been via USB cable to my Samsung Galaxy A70 android phone.

Have worked a couple of stations not knowing they had called me as I wasn’t looking at the screen all the time. An audio alert on the phone would have been most useful.

super app thanks for all your hard work

73 de Philip G0ISW
